CommandLine/libEGM/action.cpp

```diff
@@ -75,6 +75,39 @@ std::string Actions2Code(const std::vector< buffers::resources::Action >& action
   for (const auto &action : actions) {
     const auto &args = action.arguments();
 
+    std::string applies_to = action.who_name();
+    if (action.use_apply_to() && applies_to != "self") {
+      const auto &ref_as_int = [&action](const std::string &applies_to) -> std::string {
+        if (applies_to == "self") {
+          return "-1";
+        } else if (applies_to == "other") {
+          return "-2";
+        } else if (action.id() == -4) {
+          return "-100";
+        } else {
+          return std::to_string(action.id());
+        }
+      };
+
+      if (action.is_question()) {
+        if (applies_to == "other") {
+          code += "with (other) ";
+        } else if (!applies_to.empty()) {
+          code += std::string{"with ("} + ref_as_int(applies_to) + ") ";

```

Comment:
"with ("s is shorter but "with (" should also work because ref_as_in is str. I don't think ref_as_int is needed at all though.

Reply:
I have already gotten rid of this code, please check the latest commit :)